likeminds-chat-reactnative-integration/ChatSX/components/MessageInput/index.tsx

```diff
@@ -30,13 +30,15 @@ interface MessageInput {
   showJoinAlertProp: () => void;
   showRejectAlertProp: () => void;
   hintMessages?: HintMessages;
+  conversationMetaData?: any;
```

Comment:
define types. It should JSONObject?

Reply:
Made this to `Record<string, any>` as this is meta data object that we will pass